classes/fields/pick.php

```diff
@@ -903,23 +906,36 @@ public function display( $value = null, $name = null, $options = null, $pod = nu
 
 		$display_format = pods_v( static::$type . '_display_format_multi', $options, 'default' );
 
-		if ( 'non_serial' === $display_format ) {
-			$args['serial'] = false;
-		}
-
-		if ( 'custom' === $display_format ) {
-			$args['serial'] = false;
+		switch ( $display_format ) {
+			case 'ul':
+			case 'ol':
+				$value = '<' . $display_format . '><li>' . implode( '</li><li>', $value ) . '</li></' . $display_format . '>';
```

Comment:
Formatting would be nice here

```suggestion
				$value = "
					<{$display_format}>
						<li>
							" . implode( "
								</li>
								<li>
								", $value ) . "
						</li>
					</{$display_format}>
				";
```

Reply:
I wouldn't do this. As mentioned in my other comment. White spaces and `\n` tags are very annoying when it comes to inline-block elements. And especially list type elements could be set as inline block for menu's etc.